src/msw/radiobox.cpp

```diff
@@ -549,28 +537,24 @@ WX_FORWARD_STD_METHODS_TO_SUBWINDOWS(wxRadioBox, wxStaticBox, m_radioButtons)
 
 wxSize wxRadioBox::GetMaxButtonSize() const
 {
+    int cx1, cy1;
+    wxGetCharSize(m_hWnd, &cx1, &cy1, GetFont());
+
+    int radioWidth = wxRendererNative::Get().GetCheckBoxSize((wxWindow*)this).x;
```

Comment:
I'd make this `const` and also add (half) char width immediately because it's more clear than adding it later without any real explanation IMHO. And an explanation would be useful here too, in fact. It would also be better to use `const_cast`. So, all in all, I'd write this as

    // We use GetCheckBox() because there is no dedicated GetRadioBox() method
    // in wxRendererNative, but check and radio boxes are usually of the same
    // size anyhow. We also add half a character of width to account for the
    // extra space after the radio box itself.
    const int radioWidth =
        wxRendererNative::Get().GetCheckBox(const_cast<wxWindow*>(this)).x
        + GetCharWidth() / 2;